File: Java/Lista_02(Estrutura_Condicional)/Notas.java
import java.util.Locale;
import java.util.Scanner;

public class Notas {

	public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);
		Scanner s = new Scanner(System.in);
		
		System.out.print("Digite a primeira nota: ");
		double nota1 = s.nextDouble();
		
		System.out.print("Digite a segunda nota: ");
		double nota2 = s.nextDouble();
		
		double notaFinal = nota1 + nota2;
		
		System.out.printf("NOTA FINAL = %.1f\n", notaFinal);
		
		if (notaFinal<60.0) {
			System.out.println("REPROVADO");
		}
		
		s.close();

	}

}

File: C++/Lista_02(Estrutura_Condicional)/Notas.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main(){

    double nota1, nota2, notaFinal;

    cout << "Digite a primeira nota: ";
    cin >> nota1;

    cout << "Digite a segunda nota: ";
    cin >> nota2;

    notaFinal = nota1 + nota2;

    cout << "NOTA FINAL = " << fixed << setprecision(1) << notaFinal << endl;

    if (notaFinal<60.00){
        cout << "REPROVADO";
    }
    
    return 0;

}